src/app/pages/infothek/infothek-detail/infothek-detail.page.ts

```diff
@@ -26,5 +26,4 @@ export class InfothekDetailPage implements OnInit {
     selectSegment(segment: string) {
         this.selectedSegment = segment;
     }
-    ngOnInit() {}
```

Comment:
Ich würde eher den Teil aus dem Konstruktor in ngInit packen. Für den Fall, dass das Laden der Infothek mal länger benötigt.